packages/ckeditor5-typing/src/input.js

```diff
@@ -10,6 +10,7 @@
 import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
 import InsertTextCommand from './inserttextcommand';
 import InsertTextObserver from './inserttextobserver';
+import env from '@ckeditor/ckeditor5-utils/src/env';
```

Comment:
I like the comments in this file :D I spent 2 minutes scanning it thanks to them, instead of 5x more as on some other files. Just to give the perspective on why comments are important even for the review.